matrix_client/client.py

```diff
@@ -514,7 +514,7 @@ class User(object):
     """
     def __init__(self, api, user_id):
         if not (user_id.startswith("@") and user_id.count(":") == 1):
-            raise ValueError("UserID's start with @ and seperate with :")
+            raise ValueError("UserID's start with @ and separated with :")
```

Comment:
> Although the message still remains ambiguous as to what is being separated with ':', but adding extra information would possibly make it too long winded.

Yes, which was my problem. Either I separate the conditions into separate exceptions or I hope the user has a vague idea what they are doing.